test/e2e/windows/service.go

```diff
@@ -29,7 +29,7 @@ import (
 	"github.com/onsi/ginkgo"
 )
 
-var _ = SIGDescribe("Services", func() {
+var _ = SIGDescribe("[Feature:Windows] Services", func() {
```

Comment:
I think SIGDescribe already adds [sig-windows] do we need the additional tag for anything?

example:
```
[sig-windows] Services should be able to create a functioning NodePort service for Windows
```

Reply:
hiya , i just added this bc/ @marosset mentioned a similar tag addition in a previous pr :).  figure the redundant info wont hurt.  if any strong feelings happy to remove just lmk

Reply:
to make this review cleaner i just removed this , and will add that `Feature` tag in a later cleanup, if we decide we want consistency